### ISSUE TITLE
Chapters name mode - wrong titles

### ISSUE THREAD

**Issue by Kazaa3** [yermak/AudioBookConverter#52]
Since the change to 4.4.0 chapter naming works wrong. The changes in m4b title naming in AudioBookInfo.java seems to affect the chapter naming.

By naming after title, the album name is written instead the title.

The reason should be this line:
`this.setTitle(tags.get("album"));`

By naming after album, now no tag is written at all.

**Comment by yermak** [comment]
Hi,
Not sure i understand what the issue is...
Do you suggest to re-use tag album for both Series and Title?

**Comment by Kazaa3** [comment]
![titles1](https://user-images.githubusercontent.com/43350149/78112121-09b58580-73fe-11ea-9a1d-f26893951bc1.PNG)
![titles2](https://user-images.githubusercontent.com/43350149/78112125-0b7f4900-73fe-11ea-8e7a-373b8834f5d7.PNG)

**Comment by siteuser42** [comment]
Yeah the title tag seems to use album tag instead. Filename still works, so a temporary workaround could be to use mp3tag to change filenames to title tag then import it into ABC.
